tests/integration/telemetry/stats/prometheus/stats_filter_test.go

```diff
@@ -106,6 +107,11 @@ func buildQuery() (sourceQuery, destinationQuery string) {
 // proxy bootstrap config. To avoid flake, it does not verify correctness of metrics, which
 // should be covered by integration test in proxy repo.
 func TestStatsFilter(t *testing.T) {
+	if !usingOperator {
+		// TODO(bianpengyuan): remove this condition when operator is used in all tests.
```

Comment:
Can you instead do t.Skip with an attached issue?

Reply:
Switch to use t.Skip. This is not an issue. This test just should not run when using helm installation, which does not have option for stats filter. This condition could be removed when helm installation is removed completely from CI.